src/pages/public_registration/LogPanel.tsx

```diff
@@ -109,31 +115,70 @@ export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
+            <Typography>{new Date(registration.createdDate).toLocaleDateString()}</Typography>
+          </Tooltip>
+          {organizationQuery.isLoading || userQuery.isLoading ? (
+            <Skeleton sx={{ width: '4rem' }} />
+          ) : (
+            <Typography>
+              {organizationQuery.data ? organizationQuery.data?.acronym : t('common.unknown')}
+              {userQuery.data ? `, ${getFullName(userQuery.data.givenName, userQuery.data.familyName)}` : ''}
+            </Typography>
+          )}
         </StyledStatusMessageBox>
       )}
       {logs
         .sort((a, b) => new Date(a.modifiedDate).getTime() - new Date(b.modifiedDate).getTime())
         .map((logItem, index) => {
           const bgColor = logItem.type === 'PublishingRequest' ? 'publishingRequest.main' : 'doiRequest.main';
+          const modifiedDate = new Date(logItem.modifiedDate);
           return (
             <StyledStatusMessageBox key={index} sx={{ bgcolor: `${bgColor}` }}>
               <Typography>{logItem.description}</Typography>
-              <Typography>{new Date(logItem.modifiedDate).toLocaleDateString()}</Typography>
+              <Tooltip title={modifiedDate.toLocaleTimeString()} enterDelay={tooltipDelay}>
+                <Typography>{modifiedDate.toLocaleDateString()}</Typography>
+              </Tooltip>
+              {logItem.fileNames && logItem.fileNames.length > 0 && (
+                <Box
+                  component="ul"
+                  sx={{
+                    gridColumn: '1/3',
+                    m: '0',
+                    p: 'inherit',
+                    color: 'black',
+                  }}>
+                  {logItem.fileNames.map((desc, index) => (
+                    <li key={index}>
+                      <Tooltip title={desc === t('common.deleted') ? '' : desc} enterDelay={tooltipDelay}>
+                        <Typography
+                          sx={{
+                            whiteSpace: 'nowrap',
+                            overflow: 'hidden',
+                            textOverflow: 'ellipsis',
+                            fontStyle: desc === t('common.deleted') ? 'italic' : 'normal',
+                            width: 'fit-content',
+                            maxWidth: '100%',
+                          }}>
+                          {desc}
+                        </Typography>
+                      </Tooltip>
+                    </li>
+                  ))}
+                </Box>
+              )}
             </StyledStatusMessageBox>
           );
         })}
     </Box>
   );
 };
+
+function getFileNamesForTicket(ticket: PublishingTicket, registration: Registration, t: TFunction<'translation'>) {
+  return ticket.approvedFiles.map(
+    (identifier) =>
+      getAssociatedFiles(registration.associatedArtifacts).find((file) => file.identifier === identifier)?.name ??
```

Comment:
TODO: Må filtrere bort filer som ikke er av type `'PublishedFile'` for å unngå at f eks filer med administrativ avtale dukker opp i loggen.

Reply:
Mulig man heller vil returnere tre ting her: filnavn, antall slettede filer, og antall skjulte filer. Da slipper man svakheten med å string matche filer for å legge på italic styling/hover